core/common/src/main/java/alluxio/security/authentication/ImpersonationAuthenticator.java

```diff
@@ -146,4 +148,12 @@ public void authenticate(String connectionUser, String impersonationUser)
         connectionUser, impersonationUser, connectionUser, impersonationUser,
         RuntimeConstants.ALLUXIO_SECURITY_DOCS_URL));
   }
+
+  public Map<String, Set<String>> getImpersonationUsers() {
+    return Collections.unmodifiableMap(mImpersonationUsers);
+  }
+
+  public Map<String, Set<String>> getmImpersonationGroups() {
+    return Collections.unmodifiableMap(mImpersonationGroups);
+  }
```

Comment:
Setting these two to public and using this class to parse Alluxio impersonation properties